src/ConfigManagers/Omissions.php

```diff
@@ -12,4 +12,13 @@ public static function omitCredentials($credentials)
             return !in_array($key, $credentialsToOmit);
         }, ARRAY_FILTER_USE_KEY);
     }
+
+    public static function omitFields($fields)
+    {
+        $fieldsToOmit = config('auth-log.fieldsToOmit');
+
+        return array_filter($fields, function($key) use($fieldsToOmit) {
+            return !in_array($key, $fieldsToOmit);
+        }, ARRAY_FILTER_USE_KEY);
+    }
```

Comment:
Nice - I like the way you've done this to encapsulate the removal of omitted keys